tower/src/util/mod.rs

```diff
@@ -30,7 +30,7 @@ pub use self::{
     map_result::{MapResult, MapResultLayer},
     oneshot::Oneshot,
     optional::Optional,
-    ready::{ReadyAnd, ReadyOneshot},
+    ready::{Ready, ReadyAnd, ReadyOneshot},
```

Comment:
i think we need to stick an `allow(deprecated)` here...it might have to go at the top of the `use` statement though, I can't remember...
```suggestion
    #[allow(deprecated)]
    ready::{Ready, ReadyAnd, ReadyOneshot},
```